src/libs/TransactionUtils.ts

```diff
@@ -459,13 +459,16 @@ function getTagForDisplay(transaction: OnyxEntry<Transaction>, tagIndex?: number
     return getCleanedTagName(getTag(transaction, tagIndex));
 }
 
+function getCreatedDate(transaction: OnyxEntry<Transaction>): string {
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    return transaction?.modifiedCreated ? transaction.modifiedCreated : transaction?.created || '';
+}
+
```

Comment:
NAB: Could we reuse it here https://github.com/Expensify/App/blob/b12556b23729e4aa30c0f8ead5438d845ebc973a/src/libs/TransactionUtils.ts#L467